### PR TITLE
Adds a delay while welder repairing vehicles / mechs

### PR DESCRIPTION
**Pull request by ShizCalev**
## About The Pull Request

So right now there is 0 delay between welding repairs for vehicles / mecha. You can literally hop out of your murderboner durand, flip on a welder, and repair it fully from 5% HP to 100% in a split second with an autoclicker macro. This adds a delay.

## Why It's Good For The Game

Makes it so that you can't quickly hop out of your mecha/vehicle, instantly combat repair all it's damage, and hop right back into a fight. Instead, this encourages players to find safer areas before hopping out to commit to a repair.

## Changelog

:cl: ShizCalev
balance: Vehicles & Mecha now take 2.5 second to repair with a welder. 
qol: Vehicles & mecha will automatically repair over time once clicked.
qol: Vehicles & Mecha now give balloon alerts when repairing.
/:cl:

Duration is free for debate, was thinking somewhere between 2 & 3 seconds, 2.5 seemed a happy midpoint.

### CODE REVIEW

**Comment by castawaynont** [comment]
i always found it dumb that a mech with a engiborg dedicated to being a healer had absurd staying power

**Comment by trexter5555** [comment]
arent macros a bannable thing

**Comment by zxaber** [comment]
Yeah, 2.5 seconds per weld is fair enough.

**Comment by Fikou** [comment]
could you increase the amount healed or make these use while loops? (like how inducer has a charging loop of power transfering)

File: code/modules/vehicles/atv.dm [hunk 1]
```diff
@@ -68,16 +70,34 @@
...
 			turret.plane = GAME_PLANE
 
-/obj/vehicle/ridden/atv/welder_act(mob/living/user, obj/item/I)
+/obj/vehicle/ridden/atv/welder_act(mob/living/user, obj/item/W)
+	if(user.combat_mode)
+		return
+	. = TRUE
+	if(LAZYFIND(repairing_mobs, user))
+		balloon_alert(user, "you're already repairing it!")
+		return
```
Comment (Fikou) [inline code/modules/vehicles/atv.dm:79]: not needed anymore i think. mobs are limited to 1 balloon alert per object

**Pull request merged**